File: Resueltos_por_competidor/Martin_Peña/DivideAndConquer.cpp

/*
* Autor: pgms1103-afk
* Problema: 2241A - Divide and Conquer
* Juez Online: Codeforces
* Veredicto: Accepted
* URL: https://codeforces.com/problemset/problem/2241/A
*/
#include <iostream>
#include <bits/stdc++.h>
using namespace std;

int main()
{
    int t;
    cin>>t;
    for(int i = 0; i<t; i++){
        int x;
        int y;
        cin>>x;
        cin>>y;

        int resultado = 0;
        int contador = 1;
        while(resultado < x){
            resultado = y*contador;
            contador++;
        }
        if(resultado == x){
            cout<<"YES"<<endl;
        }else{
            cout<<"NO"<<endl;
        }
    }
    return 0;
}

File: Resueltos_por_competidor/Thomas_Molina/DIVIDEANDCONQUER.java
/*
 * Autor: Thomas Molina
 * Problema: A. Divide and Conquer
 * Juez online: Codeforces
 * Veredicto: Accepted
 * URL: https://codeforces.com/problemset/problem/2241/A
 */
 

import java.util.*;

public class DIVIDEANDCONQUER {
    public static void main (String[] args) 
	{
        Scanner sc = new Scanner(System.in);
        int T = sc.nextInt();
        for (int i = 0; i < T; i++){
             int X = sc.nextInt();
        int Y = sc.nextInt();
        if (X%Y==0){
            System.out.println("YES");
        }
        else{
            System.out.println("NO");
        }

        }
    }
    
}
